packages/devtools_app/lib/src/screens/deep_link_validation/validation_details_view.dart

```diff
@@ -180,15 +180,77 @@ class _DomainCheckTable extends StatelessWidget {
               ),
           ],
         ),
+        _Fingerprint(controller: controller),
         if (linkData.domainErrors.isNotEmpty)
           _DomainFixPanel(controller: controller),
         const SizedBox(height: intermediateSpacing),
-        _LocalFingerprint(controller: controller),
       ],
     );
   }
 }
 
+class _Fingerprint extends StatelessWidget {
+  const _Fingerprint({
+    required this.controller,
+  });
+
+  final DeepLinksController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    final theme = Theme.of(context);
+    return ValueListenableBuilder<String?>(
+      valueListenable: controller.localFingerprint,
+      builder: (context, localFingerprint, _) {
+        final hasPdcFingerpint =
+            controller.googlePlayFingerprintsAvailability.value;
+        final haslocalFingerpint = localFingerprint != null;
+        return ExpansionTile(
+          title: hasPdcFingerpint
+              ? const Text(
+                  'PDC fingerprint detected, enter a local fingerprint if needed',
+                )
+              : haslocalFingerpint
+                  ? const Text('Local fingerprint detected')
+                  : const _ErrorText(
+                      'Can\'t proceed check due to no fingerprint detected',
+                    ),
+          children: [
+            Padding(
+              padding: const EdgeInsets.all(largeSpacing),
+              child: RoundedOutlinedBorder(
+                child: Padding(
+                  padding: const EdgeInsets.all(largeSpacing),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      if (!hasPdcFingerpint && !haslocalFingerpint) ...[
+                        const Text(
+                          'Fix guide:',
```

Comment:
nit: if a Text widget fits on one line, no need for trailing comma. We do this for Text widgets and SizedBox. This helps make the build methods more terse